### ISSUE TITLE
lint correct - only ask y/n about practices actually found

### ISSUE THREAD

**Issue by kbjarkefur** [worldbank/stata-linter#40]
Can these questions be asked when the practice is found:

```
    Avoid using [delimit], use three forward slashes (///) instead.
    Do you want to correct this? To confirm type Y and hit enter, to abort type N and hit enter. Type
    BREAK and hit enter to stop the code. See option automatic to not be prompted before creating files.
```

It was weird to me to see this as I know I do not use `delimit` in the do-file I found. And someone who does not know what `delimit` is would be even more confused. I think the user experience would be much better if this is only prompted if it becomes relevant, and then we can say:

```
    Usage of [delimit] instead of using three forward slashes (///) was detected. See for example line: 182.
    Do you want to correct this on this line and for all other occurrences? To confirm type Y and hit enter, to abort type N and hit enter. 
    Type BREAK and hit enter to stop the code. See option automatic to not be prompted before creating files.
```

**Comment by luizaandrade** [comment]
duplicate with #9
